client/src/components/AuditBoard/Ballot.tsx

```diff
@@ -143,7 +110,7 @@ const Ballot: React.FC<IProps> = ({
 
   const [interpretations, setInterpretations] = useState<
     IBallotInterpretation[]
-  >(contests.map(emptyInterpretation))
+  >(contests.map(constructEmptyInterpretation))
```

Comment:
If I'm reading this code correctly, I think the `interpretations` state is actually unnecessary. On initial render, it's derived from `ballot.interpretations` or set to a default empty state. And then it only gets set before submitting. I think removing it might make this all even simpler.

Reply:
Tried it out to be sure I wasn't missing something: https://github.com/votingworks/arlo/pull/1463

Reply:
I know it's expanding the scope a bit but I didn't notice til just now and it feels like it might help prevent bugs like this in the future.

Reply:
Ooh interesting, taking a sec to process and incorporate 🤔

Reply:
Okay, it's clicking!

I see now that we were previously copying to state (and occasionally letting that state diverge from `ballot.interpretations`, e.g. after a user clicked submit but didn't confirm) when we could just consider `ballot.interpretations` directly (and avoid any possibility of divergence altogether)

:100: find!

https://github.com/votingworks/arlo/pull/1462/commits/135f7e02246b82c68d3c4e8f476c390c92311b13